word_guess.rb

```diff
@@ -0,0 +1,182 @@
+#IN-CLASS - THURS FEB 25
+#game similar to hangman - cat loses z if wrong guess
+#game over shows cat leaving
+
+class WordGuess
+  attr_accessor :guessed_letters, :letter, :answer, :underscores, :num, :good,
+  :cat_message, :sleepy_cat, :bike_cat, :words
+
+  def initialize
+    @words = ["loon", "unicorn", "baboon", "ocelot", "dachshund", "penguin",
+      "cuttlefish","greyhound", "meerkat", "rhinoceros", "giraffe", "slug",
+      "zonkey", "narwhal", "opossum", "emu", "wombat", "duck"].shuffle
+    @guessed_letters = [""]
+    @letter = letter
+    @answer = @words[0]
+    @underscores = underscores
+    @num = 0
+    @cat_message = cat_message
+    @sleepy_cat = sleepy_cat
+    @bike_cat = bike_cat
+  end
+
+  def if_statement
+    if @answer.delete(" ") == @underscores.join('').delete(' ')
+    puts "You won!"
+    exit
+    else
+      false
+    end
+  end
+
+  def intro
+    puts "Welcome to our game! Try to solve the word by guessing letters.
+    Once sleepy cat wakes up (loses Zs), the game is over!\n\n"
+    "word: #{underscores}\n"
+  end
+
+  def cat_message
+    cat_message = ["\nWrong letter.. Don't wake sleep cat!\n\n",
+      "\nLetter does not exist! Sleepy cat is less sleepy.\n\n",
+      "\nGuess again! Sleepy cat is stirring.\n\n",
+      "\nNot even close. Sleepy cat is about to wake up.\n\n",
+      "\nNo more guesses. Sleepy cat has decided to bike away.\n\n"]
+  end
+
+  def sleepy_cat
+    sleepy_cat = [
+    "     Z",
+    "       Z",
+    "         Z",
+    "           Z",
+    "             Z",
+    "                  |\\     _,,,---,,_
+                   /,`.-'`'    -.  ;-;;,_
+                  |,4-  ) )-,_..;\\  ( `'-'
+                 '---''(_/--'  `-'\\_)
+                 " ]
+
+  end
+
+  def bike_cat
+    bike_cat = ['
+                    ________________
+                   |      GAME      |_____    __
+                   |     OVER!!!    |     |__|  |_________
+                   |________________|     |::|  |        /
+      /\**/\       |                \.____|::|__|      <
+      ( o_o  )_    |                      \::/  \._______\
+      (u--u   \_)  |
+       (||___   )==\
+      ,dP"/b/=( /P"/b\
+      |8 || 8\=== || 8
+      `b,  ,P  `b,  ,P
+        """`     """`  ']
+  end
+
+  def new_letter
+    # prompt for very first letter guess
+    print "Enter a letter: "
+    @letter = gets.chomp
+    puts
+  end
+
+  def words_and_guessed
+    print "word: #{@underscores.join(' ')}\n"
+    print "guessed letters: #{@guessed_letters.join (' ')}\n\n"
+  end
+
+  def try_again
```

Comment:
I see some similarities in this method to the code you have written in the `new_letter` method. Any thoughts on how you could simplify this method to reuse the `new_letter` method?